docker/nvhpc/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG BASE=nvcr.io/nvidia/nvhpc:23.11-devel-cuda_multi-ubuntu22.04
```

Comment:
This image is 9.3 GB (see [NGC catalog](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nvhpc/tags))! I think it would be better to use this image `nvcr.io/nvidia/nvhpc:23.11-devel-cuda12.3-ubuntu20.04` instead, which is 5.51 GB. It would give a similar environment as the NVCC image.

Reply:
As you expected, I ran out of disk space. Try again with a smaller image.